gtsfm/runner/gtsfm_runner_base.py

```diff
@@ -167,37 +143,27 @@ def construct_scene_optimizer(self) -> SceneOptimizer:
                 logger.info("\n\nVerifier override: " + OmegaConf.to_yaml(verifier_cfg))
                 scene_optimizer.two_view_estimator._verifier: VerifierBase = instantiate(verifier_cfg.verifier)
 
+        # Override retriever.
+        if self.parsed_args.verifier_config_name is not None:
```

Comment:
why are we checking the verifier config name for this?

Reply:
I see this has been updated, so I'm taking the liberty of resolving this comment. Thanks!